lib/v2/njust/utils.js

```diff
@@ -0,0 +1,35 @@
+const delay = (timeout = 1000) => new Promise((resolve) => setTimeout(resolve, timeout));
+
+async function getContent(url, pptr = false) {
+    if (pptr) {
+        const browser = await require('@/utils/puppeteer-extra')();
+        try {
+            const page = await browser.newPage();
+            page.waitForNavigation({
+                timeout: 20000,
+            });
+            // 更改 window.navigator.webdriver 值以避开反爬
+            // await page.evaluateOnNewDocument(() => {
+            //     // eslint-disable-next-line no-undef
+            //     Object.defineProperty(navigator, 'webdriver', {
+            //         get: () => undefined,
+            //     });
+            // });
+            await page.goto(url);
+            await delay(1000);
```

Comment:
```suggestion
            await page.goto(url, {
                waitUntil: 'networkidle0',
            });
```
Using `waitUntil` option is better than waiting a fixed time.
I've also tested with `networkidle2` but it seems it's too fast for some pages and resulting in a blank page.
If you really want to pause, please use `@/utils/wait` instead of creating it again.

Reply:
done